rust/gsc/Cargo.toml

```diff
@@ -12,3 +12,5 @@ tokio = { version = "1.28.2", features = ["full"] }
 tracing = "0.1"
 tracing-subscriber = "0.3"
 axum = "0.6.18"
+serde_bytes = "0.11.9"
+blake3 = "1.4.0"
```

Comment:
Right now (but not later) the global shared cache is agnostic to the hashing algorithm that the client uses, but the clients that add jobs will have to agree with the clients that read jobs and to prevent corruption I plan on using blake3 to detect the corruption. I'll eventually migrate wake clients to use blake3 for everything as well.

In this change blake3 is just used for the hash of the request which is completely internal to the implementation and is not exposed in anyway to the client.